.changeset/sweet-shoes-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/select": patch
+---
+
+Fixing the type of SelectItemWithItems and adding labelProps.
```

Comment:
Please correct this description as well.

https://github.com/yamada-ui/yamada-ui/pull/2760/files#r1762265693

Reply:
fixed